src/charm.py

```diff
@@ -257,6 +259,14 @@ def _pebble_layer(self) -> Layer:
                     "environment": {
                         "MYSQLD_PARENT_PID": 1,
                     },
+                    "requires": [MYSQL_LOG_SERVICE],
+                    "after": [MYSQL_LOG_SERVICE],
+                },
+                MYSQL_LOG_SERVICE: {
```

Comment:
The service name is how the log will be identified in the k8s logs, e.g. the first line of:

```
2025-11-25T19:52:46.093Z [mysql] 2025-11-25T19:52:45.301296Z 11 [System] [MY-014010] [Repl] Plugin group_replication reported: 'Plugin 'group_replication' has been started.'
2025-11-25T19:52:48.945Z [pebble] GET /v1/files?action=list&path=%2Fvar%2Frun%2Fmysqld%2Fmysqld.sock&itself=true 91.795µs 200
```